xds/src/main/java/io/grpc/xds/WeightedRoundRobinLoadBalancer.java

```diff
@@ -172,8 +173,8 @@ public Subchannel createSubchannel(CreateSubchannelArgs args) {
   @VisibleForTesting
   final class WrrSubchannel extends ForwardingSubchannel {
     private final Subchannel delegate;
-    private final OrcaOobReportListener oobListener = this::onLoadReport;
-    private final OrcaPerRequestReportListener perRpcListener = this::onLoadReport;
+    private final OrcaOobReportListener oobListener = new OrcaReportListener(
```

Comment:
Here the oobListener should not be constructed like this - because it won't get the updated config for the whole lifetime of the subchannel. (as we talked before, each address update in the LB with `acceptResolvedAddresses`, some subchannels can be reused if their address is the same). 
For the optimized oobListener (avoid creating frequently), it should use the `config` object directly  in `onLoadReport`(so that it uses the updated config and it is also thread safe because `onLoadReport` is called within `syncContext`), but not constructed with the current `config.errorUtilizationPenalty` value.

Since the oobListener is set per each address update worse case, not as frequent as pick subchannel, you can just create a new ooblistener each time needed.

Reply:
Done.
Created a new OOB listener each time needed.

Thanks for the detailed context and explanation, I previously thought the OOB listener is completely separated from the lifecycle of channel & picker.